docs/dev/dev_config.md

```diff
@@ -50,22 +54,21 @@ Documented use cases that are actual configuration on a network device are consi
 
 ## New Parsers
 
-
 There are a series of considerations documented below, when developing a new parser.
 
 - Creation of a new class that must be created in `netutils/config/parser.py` file.
 - Creation of a parser class that inherits from the class `BaseConfigParser` in the Python Method Resolution Order (MRO).
-    - In nearly all cases should inherit directory off of `BaseSpaceConfigParser` or `BaseBraceConfigParser`.
```

Comment:
I don't think this is rendered correctly in mkdocs, I think this needs to be reverted.

Reply:
I'm assuming you're meaning that this particular line shouldn't be indented?

Reply:
I am asking to keep as is, I do not think it will render correctly. Under Github markdown 2 spaces is fine, mkdocs, I don't think it is. I could be wrong, but can you provide a screenshot?

Reply:
I've reverted the indent and I believe it should be as it was before with the NetScaler note added.